packages/cli/src/routeGeneration/templates/express.hbs

```diff
@@ -103,7 +103,7 @@ export function RegisterRoutes(app: Router) {
                 const controller = new {{../name}}();
               {{/if}}
 
-              templateService.apiHandler({
```

Comment:
I think we can always await, making the apiHandler return A | Promise<A>

Reply:
agree, but function must be always async: we could replace condition `{{#if ../../iocModule}}async {{/if}}`line 82 by async directly.

Reply:
Yeah, that was the ultimate goal, while still making ts not complain about it.

Reply:
> I think we can always await @jackey8616

Yes, I think that's good.

Reply:
done